apps/src/templates/projects/ProjectCard.jsx

```diff
@@ -0,0 +1,250 @@
+import React from 'react';
+import color from "../../util/color";
+import FontAwesome from '../FontAwesome';
+
+const styles = {
+  card: {
+    border: '1px solid #bbbbbb',
+    borderRadius: 2,
+    width: 215,
+    backgroundColor: color.white
+  },
+  title: {
+    paddingLeft: 15,
+    paddingRight: 10,
+    paddingTop: 10,
+    paddingBottom: 5,
+    fontSize: 16,
+    fontFamily: '"Gotham 5r", sans-serif',
+    backgroundColor: color.white,
+    color: color.gray
+  },
+  lastEdit: {
+    paddingLeft: 15,
+    paddingRight: 10,
+    paddingBottom: 10,
+    fontSize: 12,
+    fontFamily: '"Gotham", sans-serif',
+    backgroundColor: color.white,
+    color: color.gray
+  },
+  studentName: {
+    paddingLeft: 15,
+    paddingRight: 10,
+    paddingTop: 5,
+    fontSize: 12,
+    fontFamily: '"Gotham", sans-serif',
+    backgroundColor: color.white,
+    color: color.gray
+  },
+  ageRange: {
+    paddingLeft: 10,
+    paddingTop: 5,
+    fontSize: 12,
+    fontFamily: '"Gotham", sans-serif',
+    backgroundColor: color.white,
+    color: color.gray
+  },
+  firstInitial: {
+    paddingTop: 5,
+    fontSize: 12,
+    paddingLeft: 15,
+    fontFamily: '"Gotham", sans-serif',
+    backgroundColor: color.white,
+    color: color.gray
+  },
+  arrowIcon: {
+    paddingRight: 8
+  },
+  thumbnail: {
+    width: 215,
+    height: 150
+  },
+  actionBox: {
+    width: 160,
+    paddingLeft: 15,
+    paddingRight: 15,
+    border: '1px solid #bbbbbb',
+    borderRadius: 2,
+    backgroundColor: color.white,
+    boxShadow: "3px 3px 3px gray",
+    marginTop: 5,
+    position: "absolute"
+  },
+  actionText: {
+    fontSize: 12,
+    fontFamily: '"Gotham", sans-serif',
+    color: color.gray
+  },
+  delete: {
+    color: color.red,
+    borderTop: '1px solid lightGray',
+    paddingTop: 10,
+    fontSize: 12
+  },
+  xIcon: {
+    paddingRight: 5
+  },
+};
+
+const ProjectCard = React.createClass({
+  propTypes: {
+    projectData: React.PropTypes,
+    currentGallery: React.PropTypes.string,
+  },
+
+  getInitialState() {
+   return {actionsOpen: false};
+  },
+
+  dateFormatter(dateString) {
+    const date = new Date(dateString);
+    return date.toLocaleDateString();
+  },
+
+  timeFormatter(dateString) {
+    const date = new Date(dateString);
+    return date.toLocaleTimeString();
+  },
+
+  renderStudentName() {
+    // The student's name should only be visible in the classroom gallery.
+    if (this.props.currentGallery === 'class'){
```

Comment:
Rather than using bare strings like this, we might consider using an enum-like object for gallery type. An example of that is https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/code-studio/stageLockRedux.js#L13. Doing so makes it easier to understand what the expected possible values are.